cmd/controller-scaffold/cmd/crd.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"path"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+	"sigs.k8s.io/controller-tools/pkg/generate/crds"
+)
+
+var g *crds.CRDGenerator
```

Comment:
pl. try to avoid globals.

we can create a function which takes CRDGenerator as input and gives `*cobra.Command` as output which we add to the `rootCmd` below.